Cargo.toml

```diff
@@ -108,9 +107,13 @@ zeroize = ">=1.3.0, <1.6.0"
 # Make sure to remove this constraint as soon as possible.
 encoding_rs = ">=0.8.0, <=0.8.32"
 const_format = "0.2"
+tracing = "0.1"
+tracing-subscriber = "0.3"
 
 [dev-dependencies]
 toml = "0.5"
+tracing = { version = "0.1", default-features = false }
+tracing-subscriber = { version = "0.3", default-features = false, features = ["env-filter", "fmt"] }
```

Comment:
Do you still remember why we need the tracing dependencies in `dev-dependencies` as well? I haven't looked into it yet but on first sight it seems a bit strange as `env_logger` wasn't required here.

Reply:
https://crates.io/crates/test-log
<blockquote>
Note that as a user you are required to explicitly add env_logger or tracing-subscriber as a dependency to your project-under-test (when enabling the log or trace feature, respectively). E.g.,

[dev-dependencies]
env_logger = "*"
tracing = {version = "0.1", default-features = false}
tracing-subscriber = {version = "0.3", default-features = false, features = ["env-filter", "fmt"]}
</blockquote>

Reply:
According to https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html and https://doc.rust-lang.org/rust-by-example/testing/dev_dependencies.html, `dev-dependencies` are "for tests (or examples, or benchmarks) only". We already have both crates as normal `dependencies` so they shouldn't be listed in `dev-dependencies` as well. (Plus that reference is from the https://crates.io/crates/test-log crate that this PR currently doesn't use.)